src/dataframerow/utils.jl

```diff
@@ -70,36 +70,30 @@ function hashrows_col!(h::Vector{UInt},
 end
 
 # Calculate the vector of `df` rows hash values.
-function hashrows(df::AbstractDataFrame, skipmissing::Bool)
-    rhashes = zeros(UInt, nrow(df))
-    missings = fill(false, skipmissing ? nrow(df) : 0)
-    for (i, col) in enumerate(columns(df))
+function hashrows(cols::Tuple{Vararg{AbstractVector}}, skipmissing::Bool)
+    len = length(cols[1])
```

Comment:
this assumes that `cols` is not `()` which is probably the case - asking to make sure that this corner case is not relevant.

Reply:
Yes, since you can't group on no key. :-)

Reply:
Yes, but `groupby(df, [])` throws now an obscure error (`BoundsError`) - maybe some clearer message should be printed.